components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -307,8 +314,14 @@ protected void triggerNotification(User user, String type, String code, Property
         triggerNotification(user, type, code, props, null);
     }
 
+    protected void triggerNotification(User user, String type, String code, Property[] props,
```

Comment:
Why do we need to make this Protected?

Reply:
This is an overloaded method and original method is a protected method.

Reply:
I think we can make this private

Reply:
can't make this private as it is used in some other classes